libs/bjs-ecs/src/lib/ecs.spec.ts

```diff
@@ -50,4 +50,16 @@ describe('handle entities and queries', () => {
     entities = queryEntities(['enemy']);
     expect(entities.length).toBe(9);
   });
+
+  it('can listen to components added', () => {
+    function isMyEntity(isMyEntity: boolean) {
+      return { id: 'isMyEntity', isMyEntity };
+    }
+    const testQuery = ['myEntity', isMyEntity(true)];
+    entityEvents.on('add', ['myEntity', isMyEntity], (entity) => {
+      expect(entity).toHaveProperty('isMyEntity');
+    });
+
+    addEntity(testQuery);
```

Comment:
Should chek the event is actually fired, see: https://jestjs.io/docs/asynchronous#callbacks